charts/unifi/0.0.1/questions.yaml

```diff
@@ -362,4 +362,76 @@ questions:
                       schema:
                         type: string
                         default: ""
-                        
+        - variable: tcp
+          label: "TCP Ingress Configuration"
+          group: "Ingress"
+          schema:
+            type: dict
+            attrs:
+              - variable: enabled
+                label: "Enable TCP Ingress"
+                schema:
+                  type: boolean
+                  default: false
+                  show_subquestions_if: true
+                  subquestions:
+                    - variable: type
+                      label: "Select Ingress Type"
+                      schema:
+                        type: string
+                        default: "TCP"
+                        required: true
+                        editable: false
+                        enum:
+                          - value: "TCP"
+                            description: "Plain TCP Connections"
+                    - variable: entrypoint
+                      label: "Select Entrypoint"
+                      schema:
+                        type: string
+                        default: "communication-tcp"
+                        required: true
+                        enum:
+                          - value: "communication-tcp"
+                            description: "Communication-TCP: port 8080"
+                    - variable: certType
+                      label: "Select Certificate Type"
+                      schema:
+                        editable: false
+                        type: string
+                        default: ""
+                        enum:
+                          - value: ""
+                            description: "No Encryption/TLS/Certificates"
+        - variable: udp
+          label: "UDP Ingress Configuration"
+          group: "Ingress"
+          schema:
+            type: dict
+            attrs:
+              - variable: enabled
+                label: "Enable UDP Ingress"
+                schema:
+                  type: boolean
+                  default: false
+                  show_subquestions_if: true
+                  subquestions:
+                    - variable: type
+                      label: "Select Ingress Type"
+                      schema:
+                        type: string
+                        default: "UDP"
+                        required: true
+                        editable: false
+                        enum:
+                          - value: "UDP"
+                            description: "Plain UDP Connections"
+                    - variable: entrypoint
+                      label: "Select Entrypoint"
+                      schema:
+                        type: string
+                        default: "stun-udp"
+                        required: true
+                        enum:
+                          - value: "stun-udp"
```

Comment:
is this one actually called this way in traefik?

Reply:
I modeled the configuration after Deluge.

Reply:
I found the Deluge configuration before I saw your comment on using traefik. Is traefik a more correct example that I should model the configuration after?

Reply:
`entrypoint` refers to a traefik entrypoint, thats not auto created but part of the traefik configuration.

Reply:
The ingress provider we supply is traefik, so entrypoint refers to the traefik entrypoint.